tools/generator/src/BuildSettingConditional.swift

```diff
@@ -15,12 +15,12 @@ extension BuildSettingConditional {
         }
 
         var components = [key]
-        if archConditionalAllowed(on: key) {
-            components.append("[arch=\(platform.arch)]")
-        }
         if sdkConditionalAllowed(on: key) {
```

Comment:
If the order matters, do you want to add a comment stating that? It might help those of us who are helping out not to mess up the order later.🙂